src/encoded/schemas/individual_zebrafish.json

```diff
@@ -0,0 +1,116 @@
+        { "$ref": "individual.json#/properties" },
+        { "$ref": "mixins.json#/accession" },
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/modified" },
+        { "$ref": "mixins.json#/release_dates" },
+        { "$ref": "mixins.json#/documents" },
+        { "$ref": "mixins.json#/tags" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/static_embeds" }
+    ],
+    "mixinFacets" : [
+        { "$ref": "mixins.json#/facets_common" }
+    ],
+    "dependencies": {
+      "age": [ "age_units"]
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "zebrafish_life_stage": {
+            "title": "Life Stage",
+            "type": "string",
+            "lookup": 20,
+            "suggested_enum": [
+                "oogenesis",
+                "egg stage",
+                "embryonic stage",
+                "larval stage",
+                "prepupal stage",
+                "pupal stage",
+                "pharate adult stage",
+                "adult stage"
+            ]
+        },
+        "age": {
+            "title": "Age of the Individual",
+            "type": "number",
+            "exclude_from": ["FFedit-create", "submit4dn"]
+        },
+        "age_units": {
+            "title": "Age units",
+            "type": "string",
+            "exclude_from": ["FFedit-create", "submit4dn"]
+        },
+        "organism": {
+            "default": "30b53150-cff7-43ed-a8d6-fc5836df98a7",
+            "exclude_from": ["submit4dn", "FFedit-create"],
+            "lookup": 1,
+            "linkEnum": ["30b53150-cff7-43ed-a8d6-fc5836df98a7"]
+        },
+        "zebrafish_strain":{
+          "title": "Strain",
+          "lookup": 30,
+          "type": "string"
+        },
+        "zebrafish_vendor":{
+          "title": "Vendor for the Zebrafish line/strain",
+          "type": "string",
+          "linkTo": "Vendor",
+          "lookup": 40
+        },
+        "dbxrefs": {
+            "@type": "@id",
+            "rdfs:subPropertyOf": "rdfs:seeAlso",
+            "title": "External identifiers",
+            "description": "Unique identifiers from external resources.",
+            "type": "array",
+            "lookup": 1000,
+            "uniqueItems": true,
+            "items": {
+                "title": "External identifier",
+                "description": "A unique identifier from external resource.",
+                "type":  "string",
+                "pattern": "^(GEO:SAMN\\d+)$"
+            }
+        }
+    },
+    "columns": {
+        "sex": {
+            "title" : "Sex"
+        },
+        "zebrafish_life_stage": {
+            "title" : "Life Stage"
+        },
+        "zebrafish_strain": {
+            "title" : "Strain"
+        }
+    },
+    "facets": {
+        "sex": {
```

Comment:
I have no idea if sex is that relevant for zebrafish faceting for the types of experiments we are getting - do they sex the cells/embryos they are working with?

Reply:
Not sure if the lab checks the sex, and It looks like zebrafish have male and female, and also some converted ones. I will leave this as it is now, and will update when I know more. If we don't end up using sex, it will not become a facet, even if it is listed.